shell/platform/windows/flutter_window_winuwp.cc

```diff
@@ -118,7 +118,12 @@ float FlutterWindowWinUWP::GetDpiScale() {
 }
 
 bool FlutterWindowWinUWP::IsVisible() {
-  return window_.Visible();
+  // This is called from raster thread as an optimization to not wait for vsync
+  // if window is invisible. However CoreWindow is not agile so we can't call
+  // Visible() from raster thread. For now assume window is always visible.
+  // Possible solution would be to register a VisibilityChanged handler and
+  // store the visiblity state in a variable. TODO(knopp)
```

Comment:
Can you please file a bug and cross reference it in the TODO? Otherwise these TODOs tend to get lost. But otherwise LGTM.